### PR TITLE
adding episode 27

### PR DESCRIPTION
**Pull request by lpmi-13**

### CODE REVIEW

**Comment by lpmi-13** [comment]
this is mostly a placeholder while we hash out what we wanna do re: audio compression/resampling

**Comment by spikeheap** [comment]
Interesting question about file size. The current version is ~150 kbit/s, and we could drop it to 96 or 128 pretty happily.

That said, my vote is to ship this one :shipit:, and encode the next one differently. 30Mb isn't perfect, but I don't think it should block us. If we want to re-encode historical episodes we can shrink this one while we're going through the back-catalogue, ready for hockey-stick growth and things going viral 😜

File: feed.xml [hunk 1]
```diff
@@ -16,6 +16,19 @@
         <itunes:image href="https://remotehack.space/live/assets/big-hack.jpg" />
         <itunes:category text="Technology"/>
 
+        <item>
+            <title>Episode 27 - A hackside chat (AKA: airquoth the raven)</title>
```
Comment (benfoxall) [inline feed.xml:20]: # 🎤  🦅

**Review by benfoxall — APPROVED** [review]
Yeah, I reckon we should send this to the superhighway as is.  Wouldn't want to keep our loyal fanbase (@omgmog) waiting.

I'm into compressing it, but having a script that would make it repeatable/consistent seems like a good thing :100: 💅

**Comment by benfoxall** [comment]
## …wait  … …who's supposed to merge this?

I feel like there's an answer in the podcast episode, but it's not been published so I can't check.

**Comment by lpmi-13** [comment]
MERGED!!!

**Comment by omgmog** [comment]
Is anybody else hyped to listen to the latest episode???

**Comment by lpmi-13** [comment]
We have a fan! Pressure is ON!!!

**Pull request merged**